src/Angor/Client/Services/EncryptionService.cs

```diff
@@ -20,5 +24,26 @@ public async Task<string> DecryptData(string encryptedData, string password)
         {
             return await _jsRuntime.InvokeAsync<string>("decryptData", encryptedData, password);
         }
+
+        public async Task<string> EncryptNostrContentAsync(string nsec, string npub, string content)
+        {
+            var secertHex = GetSharedSecretHexWithoutPrefix(nsec, npub);
+            return await _jsRuntime.InvokeAsync<string>("encryptNostr", secertHex, content);
+        }
+
+        public async Task<string> DecryptNostrContentAsync(string nsec, string npub, string encryptedContent)
+        {
+            var secertHex = GetSharedSecretHexWithoutPrefix(nsec, npub);
+            return await _jsRuntime.InvokeAsync<string>("decryptNostr", secertHex, encryptedContent);
```

Comment:
this is not exactly wasm because we still rely on js

Reply:
It's not wasm, started the work with the wasm library and realised it's not what we need..